packages/big-design/src/components/Dropdown/Dropdown.tsx

```diff
@@ -92,25 +97,46 @@ export class Dropdown<T extends any> extends React.PureComponent<DropdownProps<T
             ref={ref}
             role="option"
           >
-            {option.type === 'link' && !option.disabled ? (
-              <Link href={option.url} target={option.target}>
-                <Flex alignItems="center" flexDirection="row">
-                  {icon && <FlexItem paddingRight="xSmall">{this.renderIcon(option, isHighlighted)}</FlexItem>}
-                  {content}
-                </Flex>
-              </Link>
-            ) : (
-              <Flex alignItems="center" flexDirection="row">
-                {icon && <FlexItem paddingRight="xSmall">{this.renderIcon(option, isHighlighted)}</FlexItem>}
-                {content}
-              </Flex>
-            )}
+            {this.getContent(option, isHighlighted)}
           </ListItem>
         );
       })
     );
   }
 
+  private getContent(option: DropdownItem<T> | DropdownLinkItem<T>, isHighlighted: boolean) {
+    const { content, disabled, icon } = option;
+    const child = (
+      <Flex alignItems="center" flexDirection="row">
+        {icon && <FlexItem paddingRight="xSmall">{this.renderIcon(option, isHighlighted)}</FlexItem>}
+        {content}
+      </Flex>
+    );
+
+    return this.getTooltip(
```

Comment:
🍹 can we rename this to something else? Seems like `getTooltip` might not return a tooltip 🤷‍♂